test/test_SUITE.erl

```diff
@@ -0,0 +1,128 @@
+                              ]}
+    ].
+
+%% -------------------------------------------------------------------
+%% Testsuite setup/teardown.
+%% -------------------------------------------------------------------
+%%
+configure_backend(Config) ->
+    rabbit_ct_helpers:merge_app_env(Config,
+                                    {rabbit, [
+                                              {auth_backends,
+                                               [rabbit_auth_backend_internal,
+                                                rabbit_auth_backend_amqp]}]}).
+
+init_per_suite(Config) ->
+    rabbit_ct_helpers:log_environment(),
+    Config1 = rabbit_ct_helpers:set_config(Config, [
+                                                    {rmq_nodename_suffix, ?MODULE}
+                                                   ]),
+
+    rabbit_ct_helpers:run_setup_steps(Config1,
+                                      [ fun configure_backend/1 ] ++
+                                      rabbit_ct_broker_helpers:setup_steps()).
+
+end_per_suite(Config) ->
+    rabbit_ct_helpers:run_teardown_steps(Config,
+                                         rabbit_ct_broker_helpers:teardown_steps()).
+
+init_per_group(_, Config) -> Config.
+
+end_per_group(_, Config) -> Config.
+
+init_per_testcase(with_backend = Testcase, Config) ->
+    PrivDir = ?config(priv_dir, Config),
+    DataDir = ?config(data_dir, Config),
+    BuildDir = filename:join([PrivDir, "build"]),
+    ok = filelib:ensure_dir(filename:join(BuildDir, "dummy")),
+    AmqpPort = rabbit_ct_broker_helpers:get_node_config(Config, 0, tcp_port_amqp),
+    Script = filename:join([DataDir, "run_backend.sh"]),
+    Port = erlang:open_port({spawn, Script}, [use_stdio,
+                                              stderr_to_stdout,
+                                              exit_status,
+                                              {env, [{"BUILD", BuildDir},
+                                                     {"AMQP_PORT", integer_to_list(AmqpPort)}
+                                                    ]}]),
+    Config1 = rabbit_ct_helpers:set_config(Config, {backend_port, Port}),
+    % wait for backend to start
+    timer:sleep(5000),
+    rabbit_ct_helpers:testcase_started(Config1, Testcase).
+
+end_per_testcase(with_backend = Testcase, Config) ->
+    Port = ?config(backend_port, Config),
+    print_port_data(Port, []),
+    {os_pid, OsPid} = erlang:port_info(Port, os_pid),
+    ct:pal("backend os pid ~p", [OsPid]),
+    KillCmd =
+        case os:type() of
+            {unix, _} -> io_lib:format("kill ~p", [OsPid]);
+            {win32, _} -> io_lib:format("taskkill /PID ~p", [OsPid])
+        end,
+    os:cmd(KillCmd),
+    true = erlang:port_close(Port),
+    rabbit_ct_helpers:testcase_finished(Config, Testcase).
+
+print_port_data(Port, Acc) ->
+    receive
+        {Port, {exit_status, X}} ->
+            ct:pal("port exited with ~p", [X]),
+            ct:pal("backend: ~p", [Acc]);
```

Comment:
You can use `"backend: ~s"` so the passed string is correctly formatted.

Currently, it prints this:
```
backend: ["gmake[1]: Entering directory '/home/dumbbell/Projects/pivotal/rabbitmq-public-umbrella/deps/rabbitmq_java_client'\nant -Dmake.bin=gmake -DUMBRELLA_AVAILABLE=true -Drabbitmqctl.bin=/home/dumbbell/Projects/pivotal/rabbitmq-public-umbrella/deps/rabbit/scripts/rabbitmqctl -Dsibling.codegen.dir=/home/dumbbell/Projects/pivotal/rabbitmq-public-umbrella/deps/rabbitmq_codegen -Dsibling.rabbitmq_test.dir=/home/dumbbell/Projects/pivotal/rabbitmq-public-umbrella/deps/rabbitmq_test build\n",
          "Buildfile: /home/dumbbell/Projects/pivotal/rabbitmq-public-umbrella/deps/rabbitmq_java_client/build.xml",
          "\n","\namqp-generate-check:\n","\namqp-generate:\n","\nbuild:\n",
          "    [javac] /home/dumbbell/Projects/pivotal/rabbitmq-public-umbrella/deps/rabbitmq_java_client/build.xml:95: warning: 'includeantruntime' was not set, defaulting to build.sysclasspath=last; set to false for repeatable builds\n",
          "    [javac] Compiling 1 source file to /home/dumbbell/Projects/pivotal/rabbitmq-public-umbrella/deps/rabbitmq_java_client/build/classes\n",
          "    [javac] warning: [options] bootstrap class path not set in conjunction with -source 1.6\n",
          "    [javac] 1 warning\n",
          "\nBUILD SUCCESSFUL\nTotal time: 1 second\n",
...
```